docs/authentication-and-authorization.md

```diff
@@ -3,66 +3,76 @@ uri: "/docs/authentication-and-authorization/"
 
-From a technical perspective, the only differences between GraphQL Queries and Mutations is the `mutation` keyword, and the GraphQL spec requires mutations to be processed synchronously, where queries can be processed Async (in environments that support it).
+Before diving into authentication and authorization, it's important to understand how GraphQL operations work in WPGraphQL.
 
-Other than that, Queries and Mutations are the same, they’re both just strings that map to functions.
+GraphQL has two main operation types:
+- **Queries**: Used for fetching data
+- **Mutations**: Used for modifying data 
 
-Now that we’re clear on Queries vs. Mutations (both are just maps to functions), authentication & authorization is left up to the application layer, not the GraphQL API layer, although some mechanisms in GraphQL can help facilitate these processes.
+While mutations use the `mutation` keyword and must be processed synchronously (one after another), both queries and mutations are fundamentally similar - they map input to resolver functions that interact with WordPress.
 
-## Authentication & Authorization
 
-- **Authentication**: the process of verifying who you are (logging in)
-- **Authorization**: the process of verifying that you have access to something – (the ability to view/change private data)
+## Authentication & Authorization Concepts
 
-## Authentication with WPGraphQL
+- **Authentication**: The process of verifying a user's identity (logging in, validating credentials)
+- **Authorization**: The process of verifying what a user can access or modify (permissions)
 
-Since WPGraphQL is a WordPress plugin that adheres largely to common WordPress practices, there are many ways to make authenticated WPGraphQL requests.
+In WPGraphQL, these processes build on WordPress's existing user and capability systems.
 
-For remote HTTP requests to the `/graphql` endpoint, existing authentication plugins *should* work fine. These plugins make use of sending data in the Headers of requests and validating the credentials and setting the user before execution of the API request is returned:
+## Authentication Methods
 
-- https://github.com/wp-graphql/wp-graphql-jwt-authentication
-- https://github.com/WP-API/Basic-Auth (even though it’s labeled for the REST API, it works well with WPGraphQL – but not recommended for non-SSL connections)
-- https://github.com/WP-API/OAuth1 (labeled for use with the WP REST API, but works well with WPGraphQL)
-- https://make.wordpress.org/core/2020/11/05/application-passwords-integration-guide/ (For WordPress 5.6 and above)
+WPGraphQL supports multiple authentication approaches depending on your use case:
 
-If the remote request is within the WordPress admin, such as the WPGraphiQL plugin, you can use the existing Auth nonce as seen in action [here](https://github.com/wp-graphql/wp-graphiql/blob/82518eafa5f383c5929111431e4a641caace3b57/assets/app/src/App.js#L58-L75).
+### 1. Remote HTTP Requests
```

Comment:
Shameless self-promotion for WPGraphQL Headless Login, which supports:
- Basic Auth
- JWT
- OAuth2,
- + the ability register custom authentication providers with PHP.

**But** it's not under the org namespace, so probably better to exclude - I just want it to be an intentional decision.

Reply:
@justlevine I'm down to include it. Do you want to write up a snippet and PR it?

Reply:
Lol was actually hoping you'd agree about excluding it - at minimum until #3049

Reply:
I'm happy to exclude for now as well. Either way is fine with me. lol

Reply:
Been excluded this long. . .no harm in excluding longer 😆

Reply:
haha perfect. Marking resolved as-is